gluoncv/model_zoo/model_zoo.py

```diff
@@ -151,6 +152,9 @@
     'yolo3_darknet53_coco': yolo3_darknet53_coco,
     'yolo3_darknet53_voc': yolo3_darknet53_voc,
     'yolo3_darknet53_custom': yolo3_darknet53_custom,
+    'yolo3_mobilenetv1_coco': yolo3_mobilenetv1_coco,
```

Comment:
to be consistent with classification model, the name should be `yolo3_mobilenet1.0_coco`,  function name is `yolo3_mobilenet1_0_coco` for example, due to the fact that function name don't support `.`